tests/integration-tests/tests/api/test_api.py

```diff
@@ -375,3 +378,99 @@ def _delete_image(region, client, image_id):
     error_message = f"No image or stack associated to parallelcluster image id {image_id}."
     with pytest.raises(NotFoundException, match=error_message):
         client.describe_image(image_id, region=region)
+
+
+@pytest.mark.skip_regions(["cn-north-1", "cn-northwest-1"])  # No Lambda container support in China regions
+def test_api_infrastructure_with_default_parameters(region, api_server_factory):
```

Comment:
we could also leave test_api_infra tests under tests/api/test_api_infrastructure.py so that we group them under the api module but we don't have too many tests in a single file